examples/restore_to_file.yaml.template

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: longhorn-restore-to-file
+  namespace: longhorn-system
+spec:
+  containers:
```

Comment:
Add `nodeName: <NODE_NAME>`  here to indicate the output should be placed on which node. Otherwise users would need to check where the pod is running to figure out where the output image is.